microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,219 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
 
 Learn more about [the icons member](https://developer.mozilla.org/docs/Web/Manifest/icons).
 
-<!-- TODO: add information about which sizes Windows require at a minimum, and which sizes are used where -->
+## Icon Image Sizes
 
+Your PWA can be enhanced on Windows by supplying the images with specific dimensions in your web app manifest. We recommend choosing one of the options below:
+
+* Level 1: (Basic image support: 512x512) This is the base image from which to generate missing images
+* Level 2: (Titles) At this level, your web app manifest contains tile images for the default (1x) display scale. Each of the images must be in PNG format and purpose any.Here is a list of recommended images and sizes.
+    * 44x44 - app icon
+    * 71x71 - small tile
+    * 150x150 - medium tile
+    * 350x150 - wide tile
+    * 310x310 - large tile
+    * 50x50 - store logo
+    * 620x300 - splash screen
+* Level 3: (Tiles with display scales) At this level, your web app manifest icons should include images with these dimensions:
```

Comment:
I suggest rephrasing to make this a bit easier to understand, by using some of your content that's further down:

```suggestion
* Level 3: (Tiles with display scales) At this level, your web app manifest should contain tile images for all Windows display scale sizes. Display scale is a user-configuration in Windows (users can change it by going to **Settings** > **Display** > **Scale**). Your web app manifest icons should include images with these dimensions:
```

Reply:
This was not addressed. I think we should do it because I find the current structure confusing. Currently, we give a bit of a description here, then a long list of icon sizes, and finally at the bottom of the list, some more description.

I suggest applying my suggested change in the previous comment, and then deleting the description at the bottom of the list.

This way, people understand upfront what level 3 is about, and can then decide to go through the list if they want.